sdk/monitor/monitor-opentelemetry/README.md

```diff
@@ -377,7 +377,7 @@ and thus draw attention to them in relevant experiences including the failures b
 
 ```typescript
 import { useAzureMonitor } from "@azure/monitor-opentelemetry";
-import { trace } from "@opentelemetry/api";
+import { trace Exception } from "@opentelemetry/api";
```

Comment:
This import looks wrong

Reply:
Missing a comma, but other than that, it's valid from the OTel api package.